internal/core/src/segcore/segment_c.cpp

```diff
@@ -78,9 +78,11 @@ Search(CSegmentInterface c_segment,
             c_placeholder_group);
         auto ctx = milvus::tracer::TraceContext{
             c_trace.traceID, c_trace.spanID, c_trace.flag};
-
-        auto span = milvus::tracer::StartSpan("SegcoreSearch", &ctx);
-
+        auto span = milvus::tracer::StartSpan("SegCoreSearch", &ctx);
+        milvus::tracer::logTraceContext(
```

Comment:
The current log level for the search is debug, which is not suitable for printing logs at the info level. The logs will explode.

Reply:
yes, I will change that